sdk/go/x/auto/test/errors/conflict_error/main.go

```diff
@@ -0,0 +1,12 @@
+package main
+
+import (
+	"github.com/pulumi/pulumi/sdk/v2/go/pulumi"
+)
+
+func main() {
+	pulumi.Run(func(ctx *pulumi.Context) error {
+		ctx.Export("exp_static", pulumi.String("foo"))
```

Comment:
Should it have a sleep or something to guarantee the clash?

Reply:
Good idea, added.